purchase_request/wizard/purchase_request_line_make_purchase_order_view.xml

```diff
@@ -33,7 +33,7 @@
                               <field name="request_id" readonly="1"/>
                               <field name="product_id" readonly="1"/>
                               <field name="name"/>
-                              <field name="product_qty"/>
+                              <field name="product_qty" readonly="1"/>
```

Comment:
@hveficent that is not correct. The person that initiates the purchase could decide to create two partial purchase orders, each with a separate quantity. The purchaser should be free to decide in the wizard.

Reply:
Considering my comment, then this PR is not required, I think

Reply:
@jbeficent It is needed then. I tested with @ageficent and if the user changes the quantity in the wizard the quotation will reflect the quantity in the purchase request line not the one in the wizard. It ignores the quantity in the wizard.

Reply:
But seems that @ageficent reversed the change, because the change now is only going to add readonly=1

Reply:
Exactly. Commit e221bfc3db749406e327a69d34ad5d2b6ea5f2ce does the fix. It is the first one in this PR.

Reply:
You are right @jbeficent, but it's still a way to limit quantity has been approved to purchase. For example, you can buy 30 when your request was on 20. I think there is need to perform some limitation like stock_quant_manual_assign module does.